test/Interop/Cxx/stdlib/foundation-and-std-module.swift

```diff
@@ -8,7 +7,7 @@
 // RUN: %target-swift-frontend %s -c -enable-experimental-cxx-interop -Xcc -std=c++17 -Xcc -fmodules-cache-path=%t -DADD_CXXSTDLIB
 // RUN: %target-swift-frontend %s -c -enable-experimental-cxx-interop -Xcc -std=c++20 -Xcc -fmodules-cache-path=%t -DADD_CXXSTDLIB
 
-// RUN: %{python} -c "import os, glob; print('\n'.join(glob.glob(os.path.join('%/t', '**', '*.pcm'), recursive=True)))" | %FileCheck %s
+// RUN: ls -R %/t | %FileCheck %s
```

Comment:
Does this work on windows?

Reply:
Hmm, good question, let's see if the Windows CI is happy about this 🤔

Reply:
Looks like the Windows CI run passed the tests